boards/msb-430/include/board.h

```diff
@@ -43,8 +43,8 @@ extern "C" {
 /**
  * @brief   Define the CPU model for the <msp430.h>
  */
-#ifndef __MSP430F1612__
-#define __MSP430F1612__
+#ifndef MSP430F1612
```

Comment:
@authmillenon  Anyway, I think I should close this PR, it seems that what I have revised here are all not include guards. :-)  Maybe I will create another PR when I catch real inappropriate include guards.